src/python/pants/__init__.py

```diff
@@ -0,0 +1 @@
+__import__("pkg_resources").declare_namespace(__name__)
```

Comment:
I considered also adding this to `pants/testutil/__init__.py`, but I don't think it's necessary. We didn't have that declaration before, and testutil worked. I think we only need it at the top level.

Reply:
We only need it in packages that are shared across multiple dists/source roots, so yeah.

Reply:
Might be worth adding a comment that this is necessary because we build two wheels under this source root. Otherwise a naive reader might wonder why it's necessary to have this when `pants` is defined only under this single source root.